src/main/scala/cognite/spark/v1/RelationConfig.scala

```diff
@@ -21,8 +21,7 @@ final case class RelationConfig(
     deleteMissingAssets: Boolean,
     subtrees: AssetSubtreeOption,
     ignoreNullFields: Boolean,
-    rawEnsureParent: Boolean,
-    enableSinglePartitionDeleteAssetHierarchy: Boolean // flag to test whether single partition helps avoid NPE in asset hierarchy builder
```

Comment:
I think we might want to keep it for potential future investigations if we see weird NPEs again, that be a way to get one of them ready

Reply:
Ah fair enough, I'll rename it to "useSinglePartition" or something